stages.html

```diff
@@ -11,36 +11,36 @@
 
   <section class="dashboard-projects">
     <div>
-      <div class="dashboard-status-definitions">
+      <div id="discovery" class="dashboard-status-definitions">
```

Comment:
it could be the way we wrote the CSS but putting the `id` on this `div` doesn't cause the browser to scroll to the right location (at least on Firefox). Maybe put these IDs on the nested `span` instead?